pyproject.toml

```diff
@@ -44,6 +44,7 @@ nidmm = {version = ">=1.4.4", extras = ["grpc"], optional = true}
 nifgen = {version = ">=1.4.4", extras = ["grpc"], optional = true}
 niscope = {version = ">=1.4.4", extras = ["grpc"], optional = true}
 niswitch = {version = ">=1.4.4", extras = ["grpc"], optional = true}
+docutils = ">=0.16, !=0.21.post1"
```

Comment:
Putting this in `[tool.poetry.dependencies]` makes the packages we upload to PyPI depend on docutils, which is unnecessary and affects clients.
 
I don't think anything outside of `[tool.poetry.group.docs.dependencies]` should depend on docutils, so move this there. Also add a comment about the workaround, because otherwise we don't need to depend on docutils at all.